### ISSUE TITLE
servant-elm-0.4.0.1 test suite failure with servant-0.13

### ISSUE THREAD

**Issue by DanBurton** [mattjbray/servant-elm#38]
As seen on the Stackage build server:

```
Failures:

  test/GenerateSpec.hs:98:
  1) encoding a simple api does it
       < generated
       > test/elm-sources/getBooksSource.elm
       7c7
       < getBooks : Bool -> Maybe (Maybe (String)) -> Maybe (Maybe (Int)) -> List (Maybe (Bool)) -> Http.Request (List (Book))
       ---
       > getBooks : Bool -> Maybe (String) -> Maybe (Int) -> List (Maybe (Bool)) -> Http.Request (List (Book))
       17c17
       <                     |> Maybe.map (toString >> Http.encodeUri >> (++) "sort=")
       ---
       >                     |> Maybe.map (Http.encodeUri >> (++) "sort=")

Randomized with seed 1283557679

Finished in 0.0027 seconds
2 examples, 1 failure
```

I was able to reproduce the failure locally like so:

```bash
stack unpack servant-elm-0.4.0.1 && cd servant-elm-0.4.0.1
edit stack.yaml # add the following stack.yaml
stack test
```

```yaml
# stack.yaml
resolver: nightly-2018-03-10
extra-deps:
- servant-0.13
- http-types-0.12.1
- servant-foreign-0.11
```

**Comment by DanBurton** [comment]
It is unclear to me whether this is a testing fluke, or a legitimate regression in servant-elm or one of its dependencies. For now, I will mark xlsx as an expected test failure on Stackage.

**Comment by mattjbray** [comment]
@DanBurton thanks, yes this was a legitimate regression. Fixed in #31 and released to Hackage as `0.5.0.0`.
